kf_lib_data_ingest/etl/configuration/log.py

```diff
@@ -10,18 +11,23 @@
     DEFAULT_LOG_OVERWRITE_OPT,
 )
 
-VERBOTEN_STRINGS = {
-    v: os.environ[v]
+VERBOTEN_PATTERNS = {
+    re.escape(os.environ[v]): f"<env['{v}']>"
     for k, v in SECRETS.__dict__.items()
     if not k.startswith("_") and v in os.environ and os.environ[v]
```

Comment:
Couldn't `v in os.environ and os.environ[v]` be `os.getenv('v')`?

Reply:
```suggestion
    if not k.startswith("_") and os.environ.get(v)
```